flink-table-store-core/src/test/java/org/apache/flink/table/store/table/WritePreemptMemoryTest.java

```diff
@@ -93,7 +93,10 @@ protected FileStoreTable createFileStoreTable(Consumer<Configuration> configure)
         Configuration conf = new Configuration();
         conf.set(CoreOptions.PATH, tablePath.toString());
         conf.set(CoreOptions.WRITE_MODE, WriteMode.CHANGE_LOG);
-        conf.set(CoreOptions.WRITE_BUFFER_SIZE, new MemorySize(30 * 1024));
+        // Run with minimal memory to ensure a more intense preempt
+        // Currently a writer needs at least one page
```

Comment:
Is there any test case that verifies whether a writer needs at lease one page or the behavior when less than a page?

Reply:
Theoretically 3 pages would run, but currently 10 are needed, and that's because the writer keeps one.